### PR TITLE
EZP-32108: Added CustomField sort clause

### PR DESCRIPTION
**Pull request by adamwojs**
> JIRA: [EZP-32108](https://jira.ez.no/browse/EZP-32108)

### Description 

Impl. visitor for https://github.com/ezsystems/ezplatform-kernel/pull/135

## Checklist

- [X] Code follows the code style of this project (use `$ composer fix-cs`).
- [x] Change requires a change to the documentation.
- [x] Code is ready for a review.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "require": {
         "php": "^7.3",
         "ext-json": "*",
-        "ezsystems/ezplatform-kernel": "^1.3@dev",
+        "ezsystems/ezplatform-kernel": "dev-ezp_32108",
```
Comment (Steveb-p) [inline composer.json:16]: Is this change intended, or just a temporary thing for tests or something?

**Review by Steveb-p — APPROVED** [review]
Add test to `CustomField` if possible.

File: composer.json (hunk 1 above)
Comment (adamwojs) [reply composer.json:16]: It's temporarily change to enable run updated integration tests suite (including test for Custom Field)

**Review by ViniTou — APPROVED** [review]
Just either mark in title that we still have tmp commit or just link green build inside PR description.

**Review by mikadamczyk — APPROVED** [review]

**Review by alongosz — APPROVED** [review]
